api/wasm/cpp/proxy_wasm_common.h

```diff
@@ -65,14 +65,16 @@ enum class HeaderMapType : int32_t {
   HttpCallResponseTrailers = 7,    // Immutable
   MAX = 7,
 };
-enum class BufferType : int32_t {
+enum class WasmBufferType : int32_t {
```

Comment:
Why not use the header files in proxy-wasm/proxy-wasm-cpp-sdk?

Reply:
I need to move all the tests to use that repo and I just haven't got around to it.

That will be my next task unless you want to do it ;)